perseo-utils/src/main/java/com/telefonica/iot/perseo/utils/DateTimeUtils.java

```diff
@@ -0,0 +1,81 @@
+package com.telefonica.iot.perseo.utils;
+
+import ca.rmen.sunrisesunset.SunriseSunset;
+import org.apache.log4j.pattern.DatePatternConverter;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+import java.util.Calendar;
+
+public class DateTimeUtils {
+
+    public static Calendar getNextSunise(Calendar day, double latitude, double longitude) {
```

Comment:
Not sure about the JavaDoc policy in this project... In Cygnus I think we use JavaDoc in the code, but not sure in Perseo Core. JavaDoc is always welcome :)

Reply:
Fixed in [0302210](https://github.com/telefonicaid/perseo-core/pull/143/commits/03022106ab35fdd3cd7aec7d76e423a429de54aa)